nodejs/install

```diff
@@ -7,7 +7,7 @@
 SOURCE_DIR=/var/lib/tsuru
 source ${SOURCE_DIR}/base/rc/config
 
-curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.7/install.sh | NVM_DIR=/etc/nvm bash
+curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.34.0/install.sh | NVM_DIR=/etc/nvm bash
```

Comment:
Can you please remove this change from this PR?

Reply:
the build of the nodejs failed then update the version.
Make sense?

Reply:
Error log
# ok 8 breaks with invalid node version

Reply:
These are different things. If you believe that's an issue you can create a separate PR that has something else todo with other than Ballerina.